src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -141,15 +141,13 @@ export function useLedgerNavigate() {
       },
 
       cancelLedgerAction() {
-        // for send ordinal '..' brings you back to the `choose-fee` step but you lose context of the ordinal
-        const backLocation = location.pathname.match(RouteUrls.SendOrdinalInscription)
-          ? RouteUrls.Home
-          : '..';
+        // Use baseUrl to determine where to go on close
```

Comment:
This is the main fix for [5143](https://github.com/leather-wallet/extension/issues/5143). 

I tried making changes to our routing to remove `/*` and restructure things but had no luck and hit old bugs from the past. 

For Ledger, we have a base route that we then append dialog routes to. 

Here, when the user rejects we take the base url from location and send them back there. 


- So if they hit: ` /sign-bip322-message` 
- Then go to `/sign-bip322-message/connect-your-ledger`
- Then `/sign-bip322-message/awaiting-approval` 
- On close we now go back to `/sign-bip322-message`